eslint/react.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
     ...reactHooksPlugin.configs.recommended.rules,
     'react/prop-types': 'off',
     'react/require-default-props': 'off',
-    'react/jsx-curly-brace-presence': ['error', 'never'],
+    'react/jsx-boolean-value': 'warn',
+    'react/jsx-curly-brace-presence': ['warn', 'as-needed'],
```

Comment:
`'as-needed'` is not documented as valid option [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md); what does it do and are we sure it's a valid configuration?

Maybe this was intended to be set for `'arrow-body-style'` instead of `'react/jsx-curly-brace-presence'`.